Documentation/InputSimulation/InputAnimationFileFormat.md

```diff
@@ -96,11 +127,11 @@ A float keyframe stores tangent and weight values alongside the basic time and v
 |---------|------|
 | Time | Float32 |
 | Value | Float32 |
-| InTangent | Float32 |
-| OutTangent | Float32 |
-| InWeight | Float32 |
-| OutWeight | Float32 |
-| WeightedMode | Int32, [Weighted Mode](#weighted-mode) |
+| InTangent | Float32 | Version 1.0 only
```

Comment:
Is this third column supposed to show up in the rendered page? I'm not sure I'm seeing them: 
![image](https://user-images.githubusercontent.com/3580640/98280611-7f637000-1f50-11eb-9843-eba43a0af59a.png)